### DISCUSSION TITLE
FR: Field instance placeholder text option

### DISCUSSION THREAD

**Discussion by MangoMarcus** [craftcms/cms#16221] [Ideas] [answered]
It doesn't seem that you can currently set placeholders at the field instance level, like you can with most field attributes.

We have a single line text field which we're reusing on lots of entry types, but it would be useful to add placeholders specific to the relevant entry types. Add the moment you need to add a UI element like a heading or a tip.

![Screenshot 2024-11-27 at 12 28 43](https://github.com/user-attachments/assets/49b77b0d-0450-45f7-b372-b6f0912cb42c)

**Comment by brandonkelly** [comment, marked as answer]
Placeholder is a Plain Text setting. It’s not currently possible to override any type-specific settings. See #8688 for a discussion about that.

**Reply by MangoMarcus** [reply]
Ahh that makes sense, ty!
